src/Symfony/Component/Mime/MimeTypes.php

```diff
@@ -2770,7 +2830,7 @@ public function guessMimeType(string $path): ?string
         'mp21' => ['application/mp21'],
         'mp2a' => ['audio/mpeg'],
         'mp3' => ['audio/mpeg', 'audio/mp3', 'audio/x-mp3', 'audio/x-mpeg', 'audio/x-mpg'],
-        'mp4' => ['video/mp4', 'video/mp4v-es', 'video/x-m4v'],
+        'mp4' => ['application/mp4', 'video/mp4', 'video/mp4v-es', 'video/x-m4v'],
```

Comment:
This one and the next one seem suspicious to me.

Reply:
application/mp4 is a registered mime type

* https://www.iana.org/assignments/media-types/application/mp4
* https://datatracker.ietf.org/doc/rfc4337/

Do you think we should add a special case for it to grant the first place to "video/mp4" (as we do for mp3 and audi) ?

https://github.com/symfony/symfony/blob/fa0a13fe0e450aba8def0b8a8362bb65ca8fc429/src/Symfony/Component/Mime/Resources/bin/update_mime_types.php#L92

Reply:
Done it / cf last comment